src/Microsoft.Diagnostics.Runtime/src/DataReaders/Core/CoreDumpReader.cs

```diff
@@ -89,44 +171,19 @@ public IList<ModuleInfo> EnumerateModules()
                 // memory range overlaps with actual modules.
                 ulong interpreter = _core.GetAuxvValue(ElfAuxvType.Base);
 
-                _modules = new List<ModuleInfo>(_core.LoadedImages.Count);
-                foreach (ElfLoadedImage img in _core.LoadedImages)
-                    if ((ulong)img.BaseAddress != interpreter && !img.Path.StartsWith("/dev"))
-                        _modules.Add(CreateModuleInfo(img));
+                _modules = new List<CoreModuleInfo>();
+                foreach (ElfLoadedImage image in _core.LoadedImages)
+                {
+                    if ((ulong)image.BaseAddress != interpreter && !image.Path.StartsWith("/dev"))
```

Comment:
This should be `!image.Path.StartsWith("/dev/")` to not ignore files in `/development` or something similar.